profiles/api/api.py

```diff
@@ -0,0 +1,54 @@
+import json
+
+from flask import Blueprint, make_response, request
+from werkzeug.exceptions import NotFound, BadRequest
+from werkzeug.wrappers import Response
+
+from profiles.exceptions import ProfileNotFound
+from profiles.repositories import Profiles
+from profiles.serializer.normalizer import normalize
+
+
+def create_blueprint(profiles: Profiles) -> Blueprint:
+    blueprint = Blueprint('api', __name__)
+
+    @blueprint.route('/profiles')
+    def _list() -> Response:
+        page = request.args.get('page', 1, type=int)
+        per_page = request.args.get('per-page', 20, type=int)
+
+        if page < 1:
+            raise BadRequest('Page less than 1')
+        elif page != request.args.get('page'):
+            raise BadRequest('Invalid page')
+
+        if per_page < 1 or per_page > 100:
+            raise BadRequest('Per page out of range')
+        elif page != request.args.get('per-page'):
+            raise BadRequest('Invalid per page')
```

Comment:
This all feels a bit verbose.

Reply:
There is probably some validation library for Flask out there, but is it worth it for a single endpoint with parameters?